pkg/reconciler/revision/resources/queue.go

```diff
@@ -63,9 +59,8 @@ var (
 
 	queueReadinessProbe = &corev1.Probe{
 		Handler: corev1.Handler{
-			HTTPGet: &corev1.HTTPGetAction{
-				Port: intstr.FromInt(networking.QueueAdminPort),
-				Path: queue.RequestQueueHealthPath,
+			Exec: &corev1.ExecAction{
+				Command: []string{"/ko-app/queue", "-probe", "true"},
```

Comment:
Can you confirm that it's our ultimate goal to:
1. Pass the user HTTP/TCP probe through for the queue to evaluate
2. Default the probe to a TCP probe

This seems like a reasonable first step, but I wanted to confirm we have a shared understanding of the end state.

Reply:
That is the plan for #4014, but I don't know that this is necessary for that.

The hope was for this to address #3308 (with the possibility of maybe using it in #4014).